DeAceroDLabs/marketplace-frontend
# Feature/dmp 491

## What is done

[[DMP-491] Auth0 login](https://deacero.atlassian.net/browse/DMP-491)

## How is done

- Added `Auth0` package.
- Added the user to `protectedroute` file.
- Created `LoadingPage` component that is used for when auth0 is loading.

## How to test

1. npm i
2. Run the project via `npm start`
3. Click on `Registrarse` to see /signup page.
4. Go back.
5. Fill with your info and click on `Ingresar`
